apps/server/src/modules/tool/external-tool/controller/api-test/tool.api.spec.ts

```diff
@@ -948,4 +955,186 @@ describe('ToolController (API)', () => {
 			});
 		});
 	});
+
+	describe('[GET] tools/external-tools/medium/:mediumId/media-source/:format/:mediaSourceId/metadata', () => {
+		describe('when mediumId, media source id and format is given', () => {
+			const setup = async () => {
+				const authEndpoint = `https://oauth-token-url.com/`;
+				const baseUrl = `https://oauth-base-url.com/`;
+				const mediaSourceEntity = mediaSourceEntityFactory.withBiloFormat({ authEndpoint, baseUrl }).build();
+				const userLicence = mediaUserLicenseEntityFactory.build({ mediaSource: mediaSourceEntity });
+
+				const { superheroUser, superheroAccount } = UserAndAccountTestFactory.buildSuperhero({}, [
+					Permission.MEDIA_SOURCE_ADMIN,
```

Comment:
Adding the permission here should not be necessary. It should be included in the buildSuperhero (default superhero permissions).